Description: Update config.* to fix FTBFS on ppc64el
   * Update config.* to fix FTBFS on ppc64el. Closes: #750693
Author: Andreas Barth <aba@ayous.org>
Bug-Debian: https://bugs.debian.org/750693

--- libevent-2.0.21-stable.orig/test/Makefile.am
+++ libevent-2.0.21-stable/test/Makefile.am
@@ -19,7 +19,11 @@ endif
 EXTRA_PROGRAMS = regress
 noinst_HEADERS = tinytest.h tinytest_macros.h regress.h tinytest_local.h
 
-TESTS = $(top_srcdir)/test/test.sh
+TESTS = test-script.sh
+test-script.sh: test.sh
+	cp $< $@
+
+DISTCLEANFILES = test-script.sh
 
 BUILT_SOURCES =
 if BUILD_REGRESS
